packages/commonwealth/client/scripts/views/pages/discussions/ThreadCard/ThreadOptions/ReactionButton/ReactionButton.tsx

```diff
@@ -90,6 +93,10 @@ export const ReactionButton = ({
       return;
     }
     if (hasReacted) {
+      if (undoUpvoteDisabled) {
+        return notifyError('Upvotes on contest entries cannot be removed');
+      }
+
```

Comment:
lets rename `undoUpvoteDisabled` to something verbose indicating its disabled coz of contents

Reply:
i added comment to explain it a bit, but I think the name is generic and this is good because this is a generic component